app/assets/javascripts/oxalis/model/reducers/save_reducer.js

```diff
@@ -24,6 +24,12 @@ function SaveReducer(state: OxalisState, action: ActionType): OxalisState {
       return state;
     }
 
+    case "EMPTY_SAVE_QUEUE": {
```

Comment:
The name "EmptySaveQueue" sounds a bit too innocent in my opinion. It's meaning might as well be understood as "push to server and empty the queue". I'd suggest "DiscardSaveQueue(Actions)" or something similar.

Reply:
✅